actions/server.search.name.in.yaml

```diff
@@ -0,0 +1,71 @@
+---
+description: Query servers
+enabled: true
+entry_point: src/server_query_actions.py
+name: server.search.name
+parameters:
+  timeout:
+    default: 5400
+  submodule:
+    default: search_servers_name_in
+    enum:
+      - "search_servers_name_in"
```

Comment:
Cant this be done with the `server.search.all` action? Since effectively we start mapping 1:1 actions:properties for each object with this approach

Reply:
yes - will be changing the `server.search.all` action to incorporate generic and string queries. I will remove this action